EpiAware/src/latent-processes.jl

```diff
@@ -1,3 +1,7 @@
+const STANDARD_RW_PRIORS =
```

Comment:
what is the logic for defining this like this? On the face of it it seems quite ugly?

Reply:
and if doing so shouldn't it have its own documentation? In R I would guess this would be a little function